deployments/engine/docker-compose/config/executor_with_s3.toml

```diff
@@ -0,0 +1,13 @@
+keepalive-ttl = "20s"
+keepalive-interval = "500ms"
+session-ttl = 20
```

Comment:
Can we use default value? don't specify them in config files

Reply:
Maybe we can solve this problem in another pr, ref: https://github.com/pingcap/tiflow/issues/7380